extensions/wikia/Helios/User.class.php

```diff
@@ -196,6 +196,22 @@ public static function clearAccessTokenCookie() {
 			time() - self::ACCESS_TOKEN_COOKIE_TTL,
 			\WebResponse::NO_COOKIE_PREFIX
 		);
+		self::clearEncryptedAccessTokenCookie( $response );
+	}
+
+	/**
+	 * Mercury's backend (Hapi) is setting access_token cookie in an encrypted form, so we need
+	 * to destroy this one as well on UserLogout
+	 * This is a temporary change which will be deleted while implementing SOC-798
+	 * @param $response
+	 */
+	public static function clearEncryptedAccessTokenCookie( $response ) {
+		$response->setcookie(
+			'sid',
```

Comment:
Please move this string to constant value.
I see almost the same call few lines above, did you consider extracting it to separate method and call with different parameters?